### PR TITLE
Update hoverText for Locations & Dungeons, and add styling for hoverText icons 

### PR DESCRIPTION
**Pull request by jhelwig**

### CODE REVIEW

**Comment by jhelwig** [comment]
This should cover the rest of #425

File: logic/poi_locations.yaml [hunk 1]
```diff
@@ -110,7 +110,7 @@
       top: 0
   logic: westOfMire
 - name: DW Death Mountain
-  hoverText: DW Death Mountain
+  hoverText: "DW Death Mountain (2) : Don't need <img src='/static/image/moonpearl.png' class='mini'>"
```
Comment (jtappa) [inline logic/poi_locations.yaml:113]: Why did you add double quotes on this one?

File: logic/poi_locations.yaml [hunk 2]
```diff
@@ -420,7 +420,7 @@
       top: 0
   logic: checkerboardCave
 - name: Hammers
-  hoverText: Hammers
+  hoverText: <img src='/static/image/hammer.png' class='mini'><img src='/static/image/hammer.png' class='mini'><img src='/static/image/hammer.png' class='mini'><img src='/static/image/hammer.png' class='mini'><img src='/static/image/hammer.png' class='mini'><img src='/static/image/hammer.png' class='mini'><img src='/static/image/hammer.png' class='mini'><img src='/static/image/hammer.png' class='mini'>!!!!!!!!
```
Comment (jtappa) [inline logic/poi_locations.yaml:423]: so many hammers

File: logic/poi_locations.yaml [hunk 3]
```diff
@@ -60,7 +60,7 @@
       top: 0
   logic: tavern
 - name: Chicken House
-  hoverText: Chicken House
+  hoverText: Chicken House <img src='/static/image/bomb.png' class='mini'>
```
Comment (jtappa) [inline logic/poi_locations.yaml:63]: is there no way to make these more dynamic so the html gets injected and you're not typing out the same thing for the bomb image over and over here in the yaml? seems not super ideal, especially if for some reason there was a directory structure or naming convention change down the line

**Review by jtappa — CHANGES_REQUESTED** [review]
Mostly just wondering about the two items that have double quotes around them and the inconsistency there, otherwise lgtm

Adding some kind of dynamic way to inject the images would be a nice to have, wouldn't necessarily have to happen in this PR.

**Comment by jtappa** [comment]
irt the dynamic images would also be in line with issue #20 customizing layout later down in the line, something you could toggle perhaps?

File: logic/poi_locations.yaml (hunk 1 above)
Comment (jhelwig) [reply logic/poi_locations.yaml:113]: The colon in the `hoverText` confuses YAML parsers if the string isn't quoted.

File: logic/poi_locations.yaml (hunk 2 above)
Comment (jhelwig) [reply logic/poi_locations.yaml:423]: One for every peg you have to smack with the hammer in that location. 😄

**Comment by jhelwig** [comment]
> 
> Adding some kind of dynamic way to inject the images would be a nice to have, wouldn't necessarily have to happen in this PR.

Some kind of templating to reduce the amount of repetition for using images in the hover text would definitely make that easier.

**Comment by jhelwig** [comment]
> 
> 
> irt the dynamic images would also be in line with issue #20 customizing layout later down in the line, something you could toggle perhaps?

The kind of customization I was thinking with #20 was more about the fixed grid that is currently used for the item listing, and the dungeon icons immediately below them (not the ones on the map. The lttp-tracker.firebaseapp.com  tracker has a mechanism for customizing which (and how many) item icons show up on each row of the grid, for both the items, and for the dungeon icons.

What kind of dynamic images are you thinking of?

**Review by jtappa — APPROVED** [review]
